app/components/new_bulk_action_button_component.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class NewBulkActionButtonComponent < ApplicationComponent
+  def link_path
+    new_bulk_action_path(search_state.reset_search.to_h.except(*bulk_action_params))
```

Comment:
Can you document what is going on here and why?

Reply:
Added a comment